NetKAN/Arkas-Config-Med-Clouds.netkan

```diff
@@ -0,0 +1,21 @@
+{
+	"spec_version"	:	1,
+	"identifier"	:	"Arkas-Config-Med-Clouds",
+	"name"		:	"Arkas Med Cloud configuration",
+	"$kref"		:	"#/ckan/kerbalstuff/1158",
+	"license"  	:	"CC-BY-NC-SA",
```

Comment:
The clouds are not licensed under `CC-BY-NC-SA`, the license is `unknown` (see LICENSE_Clouds.txt).